web/src/infra/room/GameSharing.test.tsx

```diff
@@ -45,12 +46,25 @@ describe('GameSharing', () => {
     expect(window.open as any).toHaveBeenCalled();
   });
 
-  it('should copy link', () => {
+  it('should copy link when i18n is disabled', () => {
+    const restore = mockedEnv({ NEXT_PUBLIC_I18N_ENABLED: 'false' });
     jest.useFakeTimers();
     const button = wrapper.getByLabelText('Copy');
     fireEvent.click(button);
     expect(wrapper.getByText('Copied!')).toBeInTheDocument();
     jest.runAllTimers();
     expect((window as any).copyClipboardMock).toHaveBeenCalledWith(GAME_LINK);
+    restore();
+  });
+
+  it('should copy link when i18n is enabled', () => {
```

Comment:
Wouldn't it be more simple if we get rid of this flag? It is already active in production so I think we don't need it anymore.

Reply:
Could we wait a bit longer? I'd like to be able to turn off the feature at any moment, at least until we have a game fully translated. :D

Reply:
Sure, sounds good :)